nasa_wildfires/__init__.py

```diff
@@ -1,58 +1,51 @@
 import csv
 import geojson
 import requests
+from .area import regions
```

Comment:
Lets give the `.area` file a more generic name so it can hold other option sets in the future. Maybe something like `options.py`.